workflow/controller/operator.go

```diff
@@ -262,6 +262,33 @@ func (woc *wfOperationCtx) operate() {
 		} else {
 			woc.globalParams[common.GlobalVarWorkflowStatus] = string(workflowStatus)
 		}
+
+		var failures []interface{}
```

Comment:
why not create a named struct?

Reply:
Since this is a subset of `NodeStatus`, I did not want to create an explicit new type.